### DISCUSSION TITLE
My terminal cannot load these icons

### DISCUSSION THREAD

**Discussion by LingLambda** [LazyVim/LazyVim#4809] [Q&A] [answered]
I tried using Neovim in an SSH session, and Termius displays Rust, Lua, Markdown, zzz icons, etc., properly. However, when using Tabby or MobaXterm, the icons don't display correctly. Is this due to missing fonts? Does anyone know the reason? By the way and, these icons also don't display correctly in my Chrome browser

chrome:
![image](https://github.com/user-attachments/assets/0f21a4a2-d3e0-4464-acf5-80e895bf59ec)

termuis ssh:
![image](https://github.com/user-attachments/assets/34962203-086f-4f4d-ae07-efe80d43aafc)
![image](https://github.com/user-attachments/assets/9444c1f0-e1af-49c0-8b12-801b0594c97e)

other ssh:
![image](https://github.com/user-attachments/assets/8a201d32-4774-42f6-a0d9-80135ff1cc13)
![image](https://github.com/user-attachments/assets/809117e2-c0cf-44e6-9e89-95f1534366aa)

**Comment by dpetka2001** [comment, marked as answer]
You have to have a working Nerd font installed and also configure your terminal to use that font in order for the icons to show. Read the docs for the terminal you're interested in for how to configure the fonts that it uses.

**Reply by folke** [reply]
More specifically he's using  Nerd Font, but not v3

**Comment by LingLambda** [comment]
I understand ,thank you very much for your answer



---- Replied Message ----
| From | Iordanis ***@***.***> |
| Date | 11/16/2024 18:27 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [LazyVim/LazyVim] My terminal cannot load these icons (Discussion #4809) |

You have to have a working Nerd font installed and also configure your terminal to use that font in order for the icons to show. Read the docs for the terminal you're interested in for how to configure the fonts that it uses.
